docs/backend-api/resources/tracking/tracker/rules/rule_types.md

```diff
@@ -34,63 +35,74 @@ Common parameters exist in all rule types.
 | emergency           | If `true` enables emergency notification.                                                           | boolean |
 | zone_limit_inverted | The rule tracked inside or outside zones. Default is: `false`.                                      | boolean |
 | private_rule        | Affects only sub users now. If `true` then the rule and notifications are visible only to sub user. | boolean |
+| append_zone_title   | Show or not a label of zone in a notification.                                                      | boolean |
```

Comment:
Show or not the zone labels in a notification text. Must be `null` or `false` if the `zone_limit_inverted` param set to `true`.

Reply:
done